include/ensmallen_bits/ens_version.hpp

```diff
@@ -15,17 +15,17 @@
 #define ENS_VERSION_MAJOR 2
 // The minor version is two digits so regular numerical comparisons of versions
 // work right.  The first minor version of a release is always 10.
-#define ENS_VERSION_MINOR 20
+#define ENS_VERSION_MINOR 21
```

Comment:
Yaya! The script updated the version information :)

Reply:
Yeah, very happy with the automation... now the only hard part is coming up with the release name :smile: